ecospheres/api.py

```diff
@@ -54,4 +55,4 @@ def get_bouquets(self) -> list:
         )
         data = r["data"]
         assert len(data) < LIMIT, "Too many bouquets"
-        return data
+        return [t for t in data if t["id"] != self.universe_id]
```

Comment:
I read that the wrong way and wondered why you're want bouquets not in the universe. But you're getting all bouquets in the universe (es_tag) that are not the universe itself (universe_id).

Maybe clearer naming (but couldn't find really good ones), or just a small comment?

Reply:
Ha! Clearer now?

Reply:
Somewhat :) At least that's what I also came up with thinking of a rename. Had also `es_tag` -> `universe_tag` (or `_name`). But again it's only marginally better, so up to you.